digitalocean-app-platform/README.md

```diff
@@ -0,0 +1,91 @@
+# Deploying the 1Password SCIM bridge using DigitalOcean App Platform
+This document will describe how to deploy the 1Password SCIM bridge using DigitalOcean's App Platform.
+
+## Deployment Overview
+App Platform is DigitalOcean's new, fully managed solution for deploying applications via a code repository. The idea behind App platform is that there is less of
+a focus on the manual management of containers in a given environment, and more of a focus on deploying applications for immediate access. After testing
+the deployment of the SCIM bridge with App Platform, it is now being added as a deployment option for those using DigitalOcean to host the SCIM bridge.
```

Comment:
The wording of this paragraph is slightly awkward. I would remove the part mentioning that we tested it (since that's already implicit by us officially supporting it), and the part where we describe the methodology of the App Platform itself, since it's not really our product, and we have to assume that people have read DO's marketing pages to get a fuller understanding of the service.

Reply:
Removed